README.md

```diff
@@ -1,19 +1,25 @@
-# Ubuntu Touch
+------------------------------------------------------------------------
 
-## A simple and beautiful mobile OS for everyone!
+# <a href="https://imgbb.com/"><img src="https://github.com/Lin8x/ubuntu-touch/blob/master/repo-images/uports.png" alt="uports" border="0"></a> &nbsp; Ubuntu Touch   &nbsp; <a href="https://imgbb.com/"><img src="https://github.com/Lin8x/ubuntu-touch/blob/master/repo-images/uports.png" alt="uports" border="0"></a>
```

Comment:
There's no need to link to imgbb for this image any more, the `<a>` tags can be removed.

The images can be linked to relatively, for example set the source to `repo-images/ubports.png`.

Since the images offer nothing to users with screen readers (and it'd be better if they were ignored altogether), please remove the `alt` tag and set the appropriate accessibility options, such as `aria-hidden`